### PR TITLE
Add removal of statistics#{download_counter,newest_stats} to release notes

### PR DESCRIPTION
**Pull request by dmarcoux**
Following up on #11422 and #11423. Just in case someone wonders.

### CODE REVIEW

**Comment by vpereira** [comment]
do you want to drop it as well from Release Notes?

**Comment by dmarcoux** [comment]
What do you mean the drop? Is mentioning that both endpoints have been removed not enough?

File: ReleaseNotes-2.11 [hunk 1]
```diff
@@ -75,3 +75,5 @@ Breaking API changes:
  * The API endpoint '/announcements' is deprecated and will be removed in the
    next version. Announcements can be created through Status Messages by using
    'announcement' severity. Check the API documentation about Status Messages.
+ * The API endpoints '/statistics/download_counter' and '/statistics/newest_stats'
+   have been removed.
```
Comment (hennevogel) [inline ReleaseNotes-2.11:79]:
```suggestion
 * The API endpoints '/statistics/download_counter' and '/statistics/newest_stats'
   have been removed. They have been non-functional and unused for a long time.
```

**Review by hennevogel — CHANGES_REQUESTED** [review]

**Review by hennevogel — APPROVED** [review]

**Comment by dmarcoux** [comment]
I won't wait for CI to pass since this PR is only changing release notes.

**Pull request merged**